src/nibpkg/captain.nim

```diff
@@ -0,0 +1,29 @@
+import compose
+import refmers
+import svidx
+import strformat
+import mainLookup
+import classify
+
+proc all_aboard*(variants_fn, refSeq_fn, prefix, reads_fn: string, kmerSize: int = 21, spacedSeeds : bool = false, space: int = 50, preIndex : bool = false, flank: int = 100) =
```

Comment:
rename this. :|

Reply:
Could we call it "main?" I would also suggest "run_nib?"